app/models/job.rb

```diff
@@ -1,4 +1,10 @@
 class Job < ActiveRecord::Base
-  validates :title, :level_of_interest, :city, presence: true
+  validates :title, :level_of_interest, :city, :category_id, presence: true
   belongs_to :company
+  belongs_to :category
+  has_many :comments, dependent: :destroy
+
+  def self.count_by_level_of_interest
```

Comment:
i don't see a test for this method - don't forget to test all your model methods!